### ISSUE TITLE
cloud+aerosol target classification/hydrometer type classification clarification

### ISSUE THREAD

**Issue by lisemurberg** [ACTRIS-Data-Centre/actris-vocabulary#19]
Is cloud/aerosol target classification (fra Annex 1 i DMP) the same as hydrometer type classification (ACTRIS Vocabulary)? If not, we need vocabulary for cloud/aerosol target classification. 

Need to be clarified by Ewan/Simo.

**Comment by swr99ejo** [comment]
This would depend on how we define **hydrometeor**. The classification is more extensive than just hydrometeors, as it also includes melting particles, insects

**Comment by markusfiebig** [comment]
Yes, cloud/aerosol target classification is the same as hydrometer type classification. Needs to be replaced in DMP Annex I. @lisemurberg , please update Annex I.

**Comment by lisemurberg** [comment]
Done: https://github.com/actris/data-management-plan/pull/71
